nodeup/pkg/model/crictl.go

```diff
@@ -0,0 +1,48 @@
+package model
+
+import (
+	"fmt"
+
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
+	"k8s.io/kops/util/pkg/distributions"
+)
+
+type CrictlBuilder struct {
+	*NodeupModelContext
+}
+
+var _ fi.NodeupModelBuilder = &CrictlBuilder{}
+
+func (b *CrictlBuilder) Build(c *fi.NodeupModelBuilderContext) error {
+	assetName := "crictl"
+	assetPath := ""
+	asset, err := b.Assets.Find(assetName, assetPath)
+	if err != nil {
+		return fmt.Errorf("unable to locate asset %q", err)
```

Comment:
Nit: `"unable to locate asset %q: %w", assetName, err)` I think you mean - although I'm not sure what happens if you %q an err